tests/utils/test_helpers.py

```diff
@@ -82,6 +82,14 @@ def test_jit(self, device, dtype):
         op = _torch_inverse_cast
         op_jit = torch.jit.script(op)
         assert_close(op(x), op_jit(x))
+    
+    def test_not_invertible(self, device, dtype):
+        x = torch.tensor([[0.0, 0.0], [0.0, 0.0]], device=device, dtype=dtype)
+    
+        try:
```

Comment:
the pytest way to test exceptions is via `pytest.raise` context manager
- https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions
- https://github.com/kornia/kornia/blob/main/tests/color/test_gray.py#L22

Reply:
Thank you very much. I will learn about that.

Reply:
besides, i'd like to see a unit test (with corner case scenario) proving that any of our api is affected when the situation you exposed in the original issue happens

Reply:
My apologize, I used the template to generate the description, but an irrelevant issue was automatically associated. Do i still need to write a unit test？

Reply:
ok, apologies -- i thought this PR was attempting to fix https://github.com/kornia/kornia/issues/2822 for the cross operator